cmd/wallet-web/test/fixtures/docker-compose.yml

```diff
@@ -41,3 +41,15 @@ services:
       - 443:443
     volumes:
       - ./keys/tls:/etc/sidetree/tls
+
```

Comment:
need this container for https://github.com/trustbloc/wallet/blob/main/test/fixtures/wallet-web/docker-compose-web.yml too